q2_moshpit/busco/busco.py

```diff
@@ -40,69 +123,138 @@ def evaluate_busco(
     metaeuk_rerun_parameters: str = None,
     miniprot: bool = False,
     scaffold_composition: bool = False,
-) -> None:
-    """
-    qiime2 visualization for the BUSCO assessment tool
-    <https://busco.ezlab.org/>.
-
-    Args:
-        see all possible inputs by running `qiime moshpit plot_busco`
-
-    Output:
-        plots.zip: zip file containing all of the busco plots
-        busco_output: all busco output files
-        qiime_html: html for rendering the output plots
-    """
-
-    # Create dictionary with local variables
-    # (kwargs passed to the function or their defaults) excluding
-    # "output_dir" and "bins"
+) -> pd.DataFrame:
     kwargs = {
-        k: v for k, v in locals().items() if k not in ["output_dir", "bins"]
+        k: v for k, v in locals().items() if k not in ["bins",]
     }
 
     # Filter out all kwargs that are None, False or 0.0
     common_args = _process_common_input_params(
         processing_func=_parse_busco_params, params=kwargs
     )
```

Comment:
This function could be placed in the main `evaluate_busco` function such that it is not repeatedly called. It would be enough to parse the argument once and then pass the parsed arguments to all calls of `_evaluate_busco`.